src/styles/ia-bookmarks-list.js

```diff
@@ -0,0 +1,57 @@
+import { css } from 'lit-element';
+
+export default css`
+:host {
+  display: block;
+  padding: 0 1rem;
+  color: var(--primaryTextColor);
+}
+
+small {
+  font-style: italic;
+}
+
+ul {
+  padding: 0;
```

Comment:
please make sure that scrolling to bottom allows to reach furthest text
```suggestion
  padding: 0 0 2rem 0;
```

Reply:
Thanks! Added.